srcpkgs/devilutionX/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'devilutionX'
 pkgname=devilutionX
-version=1.1.0
+version=1.2.1
 revision=1
 build_style=cmake
-configure_args="-DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
+configure_args="-DVERSION_NUM=1.2.1 -DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
```

Comment:
```suggestion
configure_args="-DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\"
 -DVERSION_NUM=$version"
```

Reply:
This new argument is necessary for compilation, but each time changes appear in the tools they compile, forcing you to change the code in the templates!    diasurgical/devilutionX#1401

Reply:
I am aware it's necessary, just moved it to a different line so the line didn't get too big.

Reply:
I understand!